checks/evaluation/permissions/permissions.go

```diff
@@ -63,25 +63,13 @@ func applyScorePolicy(results *checker.TokenPermissionsData, c *checker.CheckReq
 	dl := c.Dlogger
 	//nolint:errcheck
 	remediationMetadata, _ := remediation.New(c)
-	negativeRuleResults := map[string]bool{
+	reportedRuleResults := map[string]bool{
```

Comment:
Didn't notice in the first PR, but similar comments around necessity of initializing this map / using constants for rule name

Reply:
what do you mean?

Reply:
I think you mean creating constants to avoid repeating the string. Done.